src/index.ts

```diff
@@ -90,15 +106,20 @@ for (const service of services) {
 
 logger.debug(`Creating global index file`);
 rmSync(join(options.output, "index.ts"), { force: true });
+const generatedServices = readdirSync(options.output).filter(dir => {
```

Comment:
at the end it generate the index.ts, but if I've never generated a services, as it was before it add the export on the index, giving an error.
With this "workaround" i add in the index only the packages that export an index.ts and should be exported (so not the google package in the examples)